spec/models/mailers/review_mailer_spec.rb

```diff
@@ -71,20 +85,33 @@
     let(:friend) { create :friend, community: community, region: region }
     let!(:volunteer) { create :user, volunteer_type: 'english_speaking', community: community }
     let!(:community_admin) { create :user, :community_admin, community: community }
+    let!(:regional_admin) { create :user, :regional_admin, community: community }
 
-    subject(:mail) { ReviewMailer.changes_requested(review)}
+    subject(:mail) { ReviewMailer.changes_requested_email(review)}
 
     before do
       UserFriendAssociation.create!(friend_id: friend.id,
                                     user_id: volunteer.id)
     end
 
-    it 'emails the community admins and community volunteers associated with the friend' do
-      expect(mail.to).to eq [community_admin.email, volunteer.email]
+    it 'emails the regional admins and community volunteers associated with the friend' do
+      expect(mail.to).to eq [regional_admin.email, volunteer.email]
+    end
+
+    it 'does not email community admins' do
+      expect(mail.to).not_to eq community_admin.email
+    end
+
+    it 'generates a multipart message (plain text and html)' do
+      expect(mail.body.parts.length).to eq 2
+      expect(mail.body.parts.collect(&:content_type)).to include "text/html; charset=UTF-8"
+      expect(mail.body.parts.collect(&:content_type)).to include "text/plain; charset=UTF-8"
     end
 
     it 'renders the body' do
-      expect(mail.body.raw_source).to eq "#{friend.first_name}'s #{draft.application.category} application draft has recieved a review: #{community_friend_draft_review_url(friend.community.slug, friend, draft, review)}."
+      # expect(mail.body.encoded).to include "#{friend.first_name}'s #{draft.application.category} application draft has recieved a review: #{community_friend_draft_review_url(friend.community.slug, friend, draft, review)}."
+      expect(mail.html_part.body.raw_source).to include "#{friend.first_name}'s #{draft.application.category} application draft has recieved a review: <a href=\"#{community_friend_url(friend.community.slug, friend)}\">#{community_friend_url(friend.community.slug, friend)}</a>"
+      expect(mail.text_part.body.raw_source).to include "#{friend.first_name}'s #{draft.application.category} application draft has recieved a review: #{community_friend_url(friend.community.slug, friend)}"
```

Comment:
Not quite. They're asserting that the HTML and the plaintext versions of the email are what they should be - the content is slightly different because we're not including the `<a>` tag in the plaintext version.